src/Traces.Web/appsettings.json

```
@@ -6,5 +6,11 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
+  "Apaleo": {
+    "ClientId": "Your apaleo clientId",
```

```suggestion
    "ClientId": "Your apaleo clientId. You can define an environment variable called 'APALEO__CLIENTID'",
```

FYI: https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/index?view=aspnetcore-3.0#keys